ui/raidboss/data/06-ew/trial/endsinger.txt

```diff
@@ -0,0 +1,107 @@
+375.9 "Fatalism" sync / 1[56]:[^:]*:The Endsinger:6632:/
+388.0 "Nemesis" sync / 1[56]:[^:]*:The Endsinger:6654:/
+401.2 "--sync--" sync / 1[56]:[^:]*:The Endsinger:664A:/
+407.2 "Death's Embrace" sync / 1[56]:[^:]*:The Endsinger:6649:/
+417.3 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:6644:/
+426.4 "Hubris" sync / 1[56]:[^:]*:The Endsinger:6652:/
+436.6 "Aporrhoia" sync / 1[56]:[^:]*:The Endsinger:663D:/
+446.2 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:663E:/
+450.8 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:6644:/
+464.0 "Elegeia" sync / 1[56]:[^:]*:The Endsinger:662C:/
+483.2 "Nemesis" sync / 1[56]:[^:]*:The Endsinger:6654:/
+490.3 "--untargetable--"
+499.6 "Interstellar" sync / 1[56]:[^:]*:The Endsinger:664D:/
+501.8 "Nemesis" sync / 1[56]:[^:]*:The Endsinger:664E:/
+507.8 "--targetable--"
+513.9 "--sync--" sync / 1[56]:[^:]*:The Endsinger:664A:/
+519.9 "Death's Embrace" sync / 1[56]:[^:]*:The Endsinger:6649:/
+539.2 "Hubris" sync / 1[56]:[^:]*:The Endsinger:6652:/
+549.4 "Aporrhoia" sync / 1[56]:[^:]*:The Endsinger:663D:/
+559.0 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:663E:/
+563.5 "Elenchos" sync / 1[56]:[^:]*:The Endsinger:6644:/
+576.7 "Elegeia" sync / 1[56]:[^:]*:The Endsinger:662C:/
+
+# Add spawns
+1000.0 "Planetes" sync / 1[56]:[^:]*:The Endsinger:6B58:/ window 1000,0
+1000.0 "--untargetable--"
+1014.3 "--targetable--"
+1023.4 "Meteor Radiant" sync / 1[56]:[^:]*:Kakodaimon:6656:/
+1035.2 "Crash" sync / 1[56]:[^:]*:Kakodaimon:6657:/
+1040.2 "Crash" sync / 1[56]:[^:]*:Kakodaimon:6657:/
+1045.2 "Crash" sync / 1[56]:[^:]*:Kakodaimon:6657:/
+1050.2 "Crash" sync / 1[56]:[^:]*:Kakodaimon:6657:/
+1056.6 "Meteor Outburst" sync / 1[56]:[^:]*:Kakodaimon:665C:/
+1067.8 "Katastrophe" sync / 1[56]:[^:]*:Kakodaimon:665B:/ window 0,30
+1078.9 "Katastrophe" sync / 1[56]:[^:]*:Kakodaimon:665B:/
+1090.0 "Katastrophe" sync / 1[56]:[^:]*:Kakodaimon:665B:/
+1200.4 "Ultimate Fate" sync / 1[56]:[^:]*:The Endsinger:6B59:/ window 200,200
+1207.1 "--targetable--"
+
+# Phase 3
+2000.0 "Fatalism" sync / 1[56]:[^:]*:The Endsinger:6B5B:/ window 1000,0
+2036.0 "--untargetable--"
+2074.0 "--targetable--"
+2086.1 "Telos" sync / 1[56]:[^:]*:The Endsinger:665E:/
+2099.3 "Telomania (big)" sync / 1[56]:[^:]*:The Endsinger:665F:/ window 0,30
+2099.9 "Telomania (small)" sync / 1[56]:[^:]*:The Endsinger:665F:/
+2103.6 "Telomania (small)" sync / 1[56]:[^:]*:The Endsinger:6660:/
+2105.1 "Telomania (small)" sync / 1[56]:[^:]*:The Endsinger:6660:/
+2106.5 "Telomania (small)" sync / 1[56]:[^:]*:The Endsinger:6660:/
+2107.9 "Telomania (small)" sync / 1[56]:[^:]*:The Endsinger:6660:/
+2114.6 "Telomania (small)" sync / 1[56]:[^:]*:The Endsinger:6661:/
+2120.1 "Dead Star" sync / 1[56]:[^:]*:The Endsinger:5E4E:/
+2122.1 "Dead Star" sync / 1[56]:[^:]*:The Endsinger:5E4E:/
+2124.1 "Dead Star" sync / 1[56]:[^:]*:The Endsinger:5E4E:/
+2124.5 "Telomania (big)" sync / 1[56]:[^:]*:The Endsinger:665F:/
+2125.1 "Telomania (small)" sync / 1[56]:[^:]*:The Endsinger:665F:/
```

Comment:
The big hit at 2124.5 is probably where you want to put a `jump 2099.3`. That will give you a clean loop. You could then remove the syncs from the remaining entries to indicate that the remainder is for timeline visuals only and should never be reached.

(Additionally, you might want to build out this second fake loop by incrementing the first loop by 25.2. This will ensure the fake loop exactly matches with the main loop, and there will be no tiny shudders when the jump occurs.)